src/app/common/hooks/analytics/use-track-switch-account.ts

```diff
@@ -1,24 +1,17 @@
 import { useCallback } from 'react';
 
-import { queryClient } from '@app/common/persistence';
+import { analytics } from '@shared/utils/analytics';
 
-import { useAnalytics } from './use-analytics';
+import { queryClient } from '@app/common/persistence';
 
 export function useTrackSwitchAccount() {
-  const analytics = useAnalytics();
-
-  return useCallback(
-    async (address: string, index: number) => {
-      const accountBalanceCache = queryClient.getQueryData(['get-address-stx-balance', address]);
-      if (!accountBalanceCache) return;
-      try {
-        const hasStxBalance = !!(accountBalanceCache as any).stx.unlockedStx.amount.isGreaterThan(
-          0
-        );
-        void analytics.track('switch_account', { index, hasStxBalance });
-      } finally {
-      }
-    },
-    [analytics]
-  );
+  return useCallback(async (address: string, index: number) => {
```

Comment:
Maybe this doesn't need to be a hook anymore now?

Reply:
good spot